src/components/Layout/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Head from 'next/head';
+
+import styles from './index.module.css';
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const Layout = ({ children }: LayoutProps) => {
+  return (
+    <>
+      <Head>
+        <title>TIL</title>
+        <meta name="description" content="What Nimble has learned today" />
+        <link rel="icon" href="/favicon.ico" />
```

Comment:
Should we move this to the global `_document.page.tsx`?

```
import { Html, Head, Main, NextScript } from 'next/document';

function Document() {
  return (
    <Html lang="en">
      <Head>
        <link rel="icon" href="/favicon.ico" />
      </Head>
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  );
}

export default Document;
```

Reply:
Added in https://github.com/nimblehq/til-web/pull/19/commits/e39e645e37eb1a5805164aa818733149422375ff